src/views/TrialDescriptionPage/TrialDescriptionPage.jsx

```diff
@@ -430,6 +430,29 @@ const TrialDescriptionPage = () => {
 		return <ErrorPage initErrorsList={localState.errors} />;
 	};
 
+	const formatPrimaryPurpose = () => {
+		let formatted_purpose = false;
```

Comment:
For data type integrity, since the rest of the function sets this as a string, why not follow the same pattern here?

Reply:
Good point. Initially I was going to merge the guard in the jsx with this function but I walked it back since we're so close to launch / it's a bigger change. Updated to an empty string for now!